tests/Integration/ValidationTest.php

```diff
@@ -522,6 +522,27 @@ public function testCombinesArgumentValidationByPausingAndResuming(): void
         );
     }
 
+    public function testCombinesArgumentValidationWhenGrouped(): void
+    {
+        $this->schema .= '
+        type Mutation {
+            withMergedRules(bar: String @rules(apply: ["min:40"]) @customRules(apply: ["bool"])): User @create
+        }
+
+        type User {
+            id: ID
+            name: String
+        }
+        ';
+        $this->graphQL('
```

Comment:
```suggestion
        $this->graphQL(/* @lang GraphQL */'
```

Reply:
This isn't used anywhere. I'm simply copying codestyle from the code above.